### PR TITLE
Provide page keywords to comscore on AMP.

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?
This PR provides the page keywords to ComScore on AMP. This is already the case for non-AMP so this brings AMP into line.

## What is the value of this and can you measure success?
ComScore should be able to analyse our AMP traffic properly. We have a login for ComScore and should see it coming through on a number of dashboards.

This is what the payload now looks like:
image on http://localhost:3000/business/2017/sep/27/ryanair-cancel-flights-passengers-stansted-edinburgh?amp.

@stephanfowler can you please confirm that this is how the keywords should be passed in? On [Natalia's PR](https://github.com/guardian/frontend/pull/17818) there was use of an `extraUrlParams` key, but I can't see this in the docs I've read.

### CODE REVIEW

**Comment by JonNorman** [comment]
I'm also seeing a bunch of AMP errors when I run this, but they also appear when running `master`:
image

Are these known/accepted errors or should I clean these up as well while I'm at it?

Sincerely,
Jon 'the-AMP-noob' Norman

**Comment by JonNorman** [comment]
[Update Comscore keywords](https://trello.com/c/68l49w99/263-update-comscore-keywords)

**Comment by NataliaLKB** [comment]
@JonNorman I wouldn't worry about those errors, as they only happen locally. Since no one is actively developing on AMP it seems that the warnings have really piled up though 😞 .

I will add something to our trello to maybe look into it.

**Comment by NataliaLKB** [comment]
Code looks good, I will leave it to @stephanfowler to confirm the content 👍

File: common/app/views/fragments/amp/analytics.scala.html [hunk 1]
```diff
@@ -24,7 +24,9 @@
         {
             "vars":
                 {
-                    "c2": "6035250",
+                    "c2": "6035250"
+                },
+             "extraUrlParams": {
```
Comment (NataliaLKB) [inline common/app/views/fragments/amp/analytics.scala.html]: Ah! So this is needed then?

Comment (JonNorman) [reply common/app/views/fragments/amp/analytics.scala.html]: ¯\_(ツ)_/¯  Apparently it is! Stephan and I have landed on some common understanding and I'm going to get this merged before any of that crumbles again into chaos and confusion.

Comment (JonNorman) [reply common/app/views/fragments/amp/analytics.scala.html]: I'll wait until tomorrow to merge, just in case.

Comment (stephanfowler) [reply common/app/views/fragments/amp/analytics.scala.html]: extraUrlParams is defined in the [amp-analytics spec](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#extra-url-params). The comscore folk told us to use it for the comsecorekw property.

**Pull request merged**